### PR TITLE
fix: trigger_load can only be called once

### PR DESCRIPTION
**Pull request by BirdeeHub**
addresses issue https://github.com/nvim-neorocks/lz.n/issues/51 by adding a sane default behavior to trigger_load

### CODE REVIEW

**Comment by BirdeeHub** [comment]
that one isnt failing locally. must investigate

File: lua/lz/n/state.lua [hunk 1]
```diff
@@ -5,4 +5,7 @@ local M = {}
 ---@type table<string, lz.n.Plugin>
 M.plugins = {}
 
+---@type table<string, boolean>
```
Comment (mrcjkb) [inline lua/lz/n/state.lua:8]:
```suggestion
---@type table<string, true>
```

Comment (BirdeeHub) [reply lua/lz/n/state.lua]: didnt know you could do that

**Comment by BirdeeHub** [comment]
alright, green checkmark in like 5 mins hopefully after it finishes building neovim XD

**Comment by BirdeeHub** [comment]
yay all green and ready to go I suppose :)

**Comment by BirdeeHub** [comment]
squashed

**Comment by mrcjkb** [comment]
Closing in favour of #56

**Pull request closed without merging**